wolfSSL-FIPS-Ready/src/section01.md

```diff
@@ -0,0 +1,60 @@
+# wolfSSL FIPS Ready
+
+Do you have a project you'll need for a FIPS approved cryptographic library at 
+a future date and want to be ready for it now? wolfSSL FIPS Ready is exactly 
+what you need. It is the wolfSSL Inc FIPS enabled cryptography layer code 
+included in the wolfSSL source tree that one can enable and build. You do not 
+get a FIPS certificate, you are not FIPS approved, but you will be Ready to go 
+through the FIPS process when the time comes. FIPS Ready means that you have 
+included the FIPS code into your build and that you are operating according to 
```

Comment:
Need to be careful and specific here: "FIPS Ready means that you have included the FIPS tooling into your build, and are accessing the cryptography according to the FIPS requirements for applications."

Reply:
The point is that the application is guaranteed to be accessing the crypto per FIPS reqs.  Your phrasing says that the _tooling_ is accessing the crypto per FIPS reqs, which goes without saying.